content/building-blocks/collaborate-and-share-your-work/write-your-paper/forest-plot-generation-r.md

```diff
@@ -0,0 +1,180 @@
+data$smd <- as.numeric(data$smd)
+data$se <- as.numeric(data$se)
+
+# Forest plot
+pt <- forest(data[,c(1:3, 8:10)],
+             est = data$smd,
+             lower = data$CI_Lower, 
+             upper = data$CI_Upper,
+             sizes = 0.8,
+             is_summary = c(rep(FALSE, nrow(data)-1), TRUE),
+             ci_column = 4,
+             ref_line = 0,
+             arrow_lab = c("Favours Group 1", "Favours Group 2"),
+             xlim = c(-2, 2),
+             ticks_at = c(-2, -1, 0, 1, 2),
+             xlab = "Standardised Mean Difference",
+             theme = tm)
+
+plot(pt)
+
+
+```
+{{% /codeblock %}}
+
+
+{{% tip %}}
+Type `help(forest)` in your R terminal for more info about the `forest()` function arguments.
+{{% /tip %}}
+
+This is what the `final output` should look like:
+
+<!-- ![Forest Plot](img/forest_plot.png) -->
+
+<div style="text-align: center;">
+    <img src="../img/forest_plot.png" width="700" alt="Forest Plot">
+</div>
+
+
+## Interpret Forest Plots
+The following is an explanation of how to interpret the figure above:
+
+* **Study**: results source.
+* **Group 1/2**: number of participants in the study. Normally the two groups are split between `treatment` and `control` groups.
+* **Squares**  (red): effect size of the indvidual studies. In this example, the effect size is represented by the `standardised mean difference` between the averages of the two groups. Other possible effect sizes are `mean difference`, `odds ratio`, or `hazard ratio`.
+* **Horizontal** (blue) **lines**: `95% confidence intervals` (CI). The interpretation is that we are 95% confident that the true value of the effect size lies between the lower and upper bounds. The wider the CI the less precise the study.
+* **Diamond** (green): `pooled summary effect` of all the studies included in the meta-analysis. The middle points of the diamond represent the pooled effect, while the points on the sides its 95% confidence interval.
+* **Dotted Line**: this line is known as `line of no effect` and it is plotted at the exact point where, relative to the effect size chosen in the analyis, there is no difference between the estimates of the two groups. If the effect size is based on a difference, the line of no effect will be at 0, whereas if the effect size is based on a ratio, the line of no effect will be at 1. This line is very useful to `interpret the results`, in fact, if the CI intersects the line, the results are **NOT significant**. In this case the pooled summary effect is not significant.
+* **Weight**: study weight is proportional to study precision and it represents the influence of each individual study on the pooled effect size. More practically, when the standard error of the estimate of a study increases, its weight decreases. An alternative approach is to use a weight that is positively correlated with sample size.
+* **SMD (95% CI)**: summary of effect size and confidence intervals in the graphical representation.
```

Comment:
nice job overall -- add a sentence in the overview along the lines of "in this guide you will learn how to..." and also a summary, similar to e.g https://tilburgsciencehub.com/building-blocks/automate-and-execute-your-work/reproducible-work/mem-storage-gcp/